src/components/molecules/products.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+const Products = (products) => (
```

Comment:
Comment ça se fait que ça fonctionne ?
```suggestion
const Products = ({ products }) => (
```

Reply:
mdrrr j'avoue zéro idée pq ca fonctionne j'avais pas tilté, mais ca fonctionne